src/main/java/io/lettuce/core/output/ArrayComplexData.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2024, Redis Ltd. and Contributors
+ * All rights reserved.
+ *
+ * Licensed under the MIT License.
+ *
+ * This file contains contributions from third-party contributors
```

Comment:
You don't need Apache 2.0 license note in a brand-new file :) Everything after "Licensed under the MIT License." can be safely removed.